### ISSUE TITLE
Daily report for 03-18-2020 includes US, US error.

### ISSUE THREAD

**Issue by plscks** [CSSEGISandData/COVID-19#999]
Daily report commit 7e34a979ea2646a688320d06964aba016fb8028c includes `US,US,2020-03-18T19:33:03,1,0,106,37.0902,-95.7129` at line 271 Which represents 1 Confirmed, 0 Deaths, and 106 Recovered which translates to -105 active cases there at [https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6](url)

I believe this to be an error.

**Comment by jnolan00** [comment]
Agreed. Not sure if all the recovered got aggregated together or what but the recovered count is missing for all the states in the US as well.

**Comment by arik-so** [comment]
The coordinates used for the US entry place all 106 of the recoveries in Kansas.

**Comment by eyal318** [comment]
Looks like in Washington state and Nevada in the US, the total number of confirmed dropped day-over-day, which also looks a little off.

**Comment by plscks** [comment]
The coordinates of the US, US entry are about spot on with where Google Maps leads you if you look for directions to the United States.

**Comment by congeo** [comment]
Time series data for Recovered counts (time_series_19-covid-Recovered.csv) are showing all zero's for the US for 2020-03-18.  values for 2020-03-17 and earlier seem OK.

**Comment by ryanwoconnor** [comment]
Same as #1082 and #1075. Can we please get a fix?
